### ISSUE TITLE
Prompts and labels for reproducing classifier results of section 7

### ISSUE THREAD

**Issue by itsmemala** [javiferran/sae_entities#3]
Hi, could you help me locate the **model prompts** (train,val,test), **answers** and the **answer labels** needed for reproducing the following from section 7 of the paper:
"Using this latent as a classifier, it achieves 73.2 AUROC score, and by calibrating the decision threshold on a validation set, it
gets an F1 score of 72"

I believe this result is for Gemma 2B IT?
